INSTALL.rst

```diff
@@ -12,7 +12,7 @@ provided with the source.
 Dependencies
 ************
 
-A full list of the Cyclus package dependencies is listed :doc:`here <DEPENDENCIES>`:
+A full list of the Cyclus package dependencies is listed `here`_.
```

Comment:
Have you tried building the website with this change? This is what I get when I click on the link "here".

<img width="871" alt="Screenshot 2024-04-01 at 6 34 38 PM" src="https://github.com/cyclus/cyclus/assets/79653949/94c87915-5a97-48f0-a88a-53e07a6418e7">

Reply:
I did test this with a local build and it works for the instance below. I don't think this section shows up in the website, the `.. website_include_start` is after this section. Which page of the website does this link show up in?

Reply:
The Cyclus Installation Guide (/user/CYCLUS_INSTALL.html).  I am building it with these modifications to the Makefile so that it fetches your new files.

<img width="871" alt="Screenshot 2024-04-02 at 9 13 01 AM" src="https://github.com/cyclus/cyclus/assets/79653949/98fc09ff-7303-465f-a548-70b6a256b916">

Reply:
See my comment in the main conversation.